wolfcrypt/src/ecc.c

```diff
@@ -228,8 +228,8 @@ ECC Curve Sizes:
 #define MAX_ECC_BITS_USE    MAX_ECC_BITS_NEEDED
 #endif
 #define ECC_KEY_MAX_BITS(key)                                       \
-    (((key == NULL) || (key->dp == NULL)) ? MAX_ECC_BITS_USE        \
-                                          : (key->dp->size * 8))
+    ((((key) == NULL) || ((key)->dp == NULL)) ? MAX_ECC_BITS_USE    \
+                                          : (unsigned)((key)->dp->size * 8))
```

Comment:
I prefer we avoid the shorthand unsigned and be explicit with 'unsigned int' or other type desired

Reply:
I don't like it either, but the incumbent code uses the bare `unsigned`, and they'd all need to be changed to be sure we aren't adding problems on weird targets where `unsigned` isn't `unsigned int`.  didn't seem worth the greater disruption to change them all.